ats/job_application/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class JobApplicationConfig(AppConfig):
+    name = 'job_application'
```

Comment:
any app must be plural not singular `job_applications`, I prefer to rename it to `applications`